prow/config.yaml

```diff
@@ -257,6 +257,19 @@ tide:
     - do-not-merge/invalid-owners-file
     - do-not-merge/work-in-progress
     - needs-ok-to-test
+  - repos:
+    - kubernetes-incubator/service-catalog
+    labels:
+    - lgtm
+    - approved
+    - "cncf-cla: yes"
+    - LGTM1
+    - LGTM2
```

Comment:
I figure this set of labels means we have to do both flows for a bit.

Reply:
yeah this would require all of these labels.

Reply:
excellent. a good start.

Reply:
Is having both the lgtm and approved and LGTM1/2 an interim thing? Like once we move all the way to prow I'm hoping we can stop using those custom labels. 😀

Reply:
for now. My intention is to make sure we don't accidentally merge anythings that are not reviewed enough right now. WDYT?

Reply:
At the moment we aren't quite following how to apply /lgtm and /approve the same way the regular k8s orgs do so I think it's good to keep the 1/2 labels in the mix as well until we are ready to take off the training wheels.